### PR TITLE
chore: Show tag usage when deleting

### LINKED ISSUE #3489: UX Audit - Show how many features are using a tag when deleting
### Is your feature request related to a problem? Please describe.

When deleting a tag that’s actively being used, it would be nice to know it’s actively in use and how many flags will be affected by deleting it.

### Describe the solution you'd like.

In the confirmation delete modal, show how many features are tagged

### Describe alternatives you've considered

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

Shows when a tag is in use when attempting to delete.

image

## How did you test this code?

Tried deleting our tags

### CODE REVIEW

**Review by novakzaballa — COMMENTED** [review]
LGTM. However, for example, if I have 2 flags with the same tag, and I remove it from one, the message still says that 2 features are using the tag.

Screen Shot 2024-03-06 at 14 41 33

Screen Shot 2024-03-06 at 14 41 45

**Comment by kyle-ssg** [comment]
Thanks for raising @novakzaballa , this is fixed

File: frontend/web/components/TagUsage.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  )
+  if (!data?.count) {
+    return null
+  }
+  return (
+    <div className='mt-4'>
+      <WarningMessage
+        warningMessage={`${data?.count} feature${
+          data?.count !== 1 ? 's' : ''
+        } are using this tag.`}
```
Comment (matthewelwell) [inline frontend/web/components/TagUsage.tsx]:
```suggestion
        warningMessage={`${data?.count} feature${
          data?.count !== 1 ? 's are' : ' is'
        } using this tag.`}
```

**Review by matthewelwell — CHANGES_REQUESTED** [review]
image

🤦

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**